e2e-tests/commands/check-buyer-country-form.js

```diff
@@ -84,7 +84,8 @@ const checkSubmitButton = () => {
 };
 
 const checkValidationErrors = () => {
-  partials.errorSummaryListItems().should('exist');
+  cy.checkErrorSummaryListHeading();
+
   partials.errorSummaryListItems().should('have.length', 1);
```

Comment:
what about the function we have to check errors including summary list and field errors all in one?

Reply:
Do you mean `submitAndAssertFieldErrors`? I did think of this, it'd be a pretty epic job, there is a ticket in the backlog for this, EMS-969.